src/platform/web.rs

```diff
@@ -248,6 +248,8 @@ pub trait CustomCursorExtWebSys {
     /// It uses the [url css function](https://developer.mozilla.org/en-US/docs/Web/CSS/url),
     /// but browser support for image formats is inconsistent. Using [PNG] is recommended.
     ///
+    /// The alpha channel is assumed to be **not** premultiplied.
+    ///
```

Comment:
```suggestion
```
Not relevant here, because we don't parse the image, we pass it straight on as a URL to CSS.

Reply:
Hm, but you set that alpha is not premultiplied or you just pass as is when it's loaded like that?

Reply:
When it's loaded from a URL it bypasses that part of the code indeed.